### DISCUSSION TITLE
Cannot delete locations

### DISCUSSION THREAD

**Discussion by gpsdev1** [snipe/snipe-it#12237] [Q&A]
There does not seem to be an ability to delete locations in SnipeIT (6.0.13, build 8923). Bulk editing locations seems to be an oft-requested feature enhancement, but there actually does not appear to be any feature to delete a location at all. Drilling to the location allows me to Update Location, but even within that screen there is no delete function.

Am I missing something? Can you even delete a location? Thank you!

**Comment by inietov** [comment]
From this screen you can delete them if you don't have assets or users associated to them, with the button that have a trash can icon. 

![image](https://user-images.githubusercontent.com/653557/206735628-01a36ee3-9416-4e79-857f-a07ff7042a2a.png)

**Comment by gpsdev1** [comment]
Thank you, I now see the individual delete icons. SnipeIT doesn't resize windows so I need to scroll to the bottom of the entire page to scroll right, then scroll back up to find the rows I want to delete.
